### ISSUE TITLE
why the labels are the same as the inputids in SFTDataset?

### ISSUE THREAD

**Issue by RankKCodeTalker** [hpcaitech/ColossalAI#3406]
why the labels are the same as the input_ids in SFTDataset? I think SFT should be trained only on crossentropyloss of lmheadmodel output logits of labels part and labels.

**Comment by JThh** [comment]
As per how [Cross Entropy Loss](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html) is defined!
